lib/aida/bot.ex

```diff
@@ -75,10 +75,12 @@ defmodule Aida.Bot do
       message
       |> reset_language_if_invalid()
       |> choose_language_for_single_language_bot()
+      |> set_session_do_not_disturb()
       |> handle()
       |> greet_if_no_response_and_language_was_set(message)
       |> unset_session_new
       |> log_incoming
+      |> clean_reply_if_do_not_disturb
```

Comment:
I don't like having to handle this case separately from the rest of the skills and then having to clear the other skills response. What we could do is including the front desk at the skill level for the handle, leaving it the chance to say that the message is for him in the same handle workflow, so the answer would be only that of the front desk.